src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -479,9 +484,10 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
     case Event(message @ CallFailedNonRetryable(callKey, events, returnCode, failure), data) if data.isPersistedRunning(callKey) =>
       persistStatusThenAck(callKey, ExecutionStatus.Failed, sender(), message, callOutputs = None, returnCode = returnCode)
       val updatedData = data.addPersisting(callKey, ExecutionStatus.Failed)
+      addCallFailureEvent(callKey.toDatabaseKey, failure)
```

Comment:
Sure, if the message in the failure is that nice one with the ordinals and retry limits.  :smile: